MilitaryManager.Attachments.API/Controllers/AttachmentsController.cs

```diff
@@ -36,29 +38,35 @@ public FileStreamResult GetDocument([FromQuery] string name)
             return new FileStreamResult(fileStream, "application/pdf");
         }
 
-        [HttpGet]
+        // GET: GetTemplates
+
+        [HttpPost]
         [Route("generate")]
```

Comment:
The path is incorrect. We cannot use this implementation in Angular. Change it to [Route("{id}")] or don't specify the path at all. To understand how we use controllers look at Angular base.services.ts lines 81-87, 106-144

Reply:
This path was by default from the beginning